nipype/algorithms/rapidart.py

```diff
@@ -665,14 +534,122 @@ def _run_interface(self, runtime):
                 nrows.append(mc_in.shape[0])
             matrix = self._get_spm_submatrix(spmmat, sessidx, rows)
             self._stimcorr_core(motparamlist[i], intensityfiles[i],
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Too many positional arguments for method call](https://www.codacy.com/app/krzysztof-gorgolewski/nipy_nipype/file/2047345374/issues/source?bid=2767207&fileBranchId=2890884#l536)